### PR TITLE
C1 beshoy intrested

### PR DESCRIPTION
**Pull request by BeshoyAtef**
@mohamedahmedawad93 @Tharwat7 Please review  my code of fixing the intrested in

### CODE REVIEW

**Comment by BeshoyAtef** [comment]
Sprint Zero

File: project/tager/tager_www/views.py [hunk 1]
```diff
@@ -816,72 +816,19 @@ def facebook_login_done(request):
...
-#         for a_post in post:
-#             post_obj.append(Post.objects.get(id = a_post))
-#         if not post_obj:
-#             return HttpResponse("there is no posts with these values please refine your search.")
-
-#         else:
-#             print post_obj
-#             post_list=filter_posts(post_obj)
-#             return render('main.html', {'post_list' : post_list})
+#Beshoy intrested method Takes a request
```
Comment (Tharwat7) [inline project/tager/tager_www/views.py:819]: @BeshoSmile 
Please add a little more detial to the method documentation regarding what it does :)

**Comment by BeshoyAtef** [comment]
@AbdelrahmanAyad

**Comment by Reem-Sharkawy** [comment]
is this pull request merged or not yet ?!
3shan el master is not running for the sake of interested !

**Comment by ahmedshireef** [comment]
Comment the url containing interested in and u r clear to go

Sent from my BlackBerry® smartphone from Etisalat Misr

-----Original Message-----

**Comment by HalaKamal** [comment]
tage me when u finish the diagram  @BeshoSmile

**Comment by HalaKamal** [comment]
@BeshoSmile  i cant see the uml where ?

**Comment by HalaKamal** [comment]
@BeshoSmile  if your uml called interested the render must be doted and ul in the post.html must be in templates not ul and there must be a render to the actor when the story finishes

**Comment by HalaKamal** [comment]
@BeshoSmile  uml reopened

**Comment by BeshoyAtef** [comment]
the story doest render to the actor it renders to the html (only change a
color )

**Comment by BeshoyAtef** [comment]
doted lines only when return to user sa7 ?

**Comment by HalaKamal** [comment]
okrender a doted line to the user and write on it colour changed

**Comment by HalaKamal** [comment]
say specifically what happened the colour of ... wll be changed

**Comment by HalaKamal** [comment]
and make the lif time of the actor more longer than others

**Comment by HalaKamal** [comment]
change controller to tager_www

**Comment by HalaKamal** [comment]
if there any condition in the view that it will make if or loop please state it on it

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=vfpLRidLUUGCpzu_3Y3Vk5IMPspzJ3u-ouytYU2O0lc.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=3e29f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy Atef wants to connect with you.

<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=vfpLRidLUUGCpzu_3Y3Vk5IMPspzJ3u-ouytYU2O0lc.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=3e29f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=ProfileThumb>

Beshoy Atef
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=vfpLRidLUUGCpzu_3Y3Vk5IMPspzJ3u-ouytYU2O0lc.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=3e29f5ef-7d
c8-e211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInv
ite&utm_medium=Email&sendercc=EG&emailLinkType=ProfileName>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=vfpLRidLUUGCpzu_3Y3Vk5IMPspzJ3u-ouytYU2O0lc.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIzIn0&cnId=88470b98-4248-e011-88
9f-c965259bf186&emailMessageId=3e29f5ef-7dc8-e211-a4c9-c701ce909180&emailType=Inv
ite&sdate=201305291636&utm_source=EmailInvite&utm_medium=Email&sendercc=EG&emailL
inkType=MainCallToAction>

reply+i-13574513-077aa914744a1748392fe83a8ef5edfd0acffc6a-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=vIKfMygaTYc8Xv6
hUTSO5t3tw4NImz9VA1i6UtgKtLk.eyJlbWFpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRh
MTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzY5ODQ1NDA2IiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=3e29f5ef-7dc8-e
211-a4c9-c701ce909180&emailType=Invite&sdate=201305291636&utm_source=EmailInvite&
utm_medium=Email&sendercc=EG&emailLinkType=Unsubscribe> .

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=Awyk_ENG2JjVVLMZL35lXZgVNfKVdBik-cdYXe_prRs.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODQ4IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=fac8fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050024&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy is waiting for you to accept this pending invitation.

<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=Awyk_ENG2JjVVLMZL35lXZgVNfKVdBik-cdYXe_prRs.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODQ4IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=fac8fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050024&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=ProfileT
humb>

Beshoy Atef
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=Awyk_ENG2JjVVLMZL35lXZgVNfKVdBik-cdYXe_prRs.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODQ4IiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=fac8fa14-73
cd-e211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050024&utm_so
urce=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=ProfileN
ame>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftuf/emailinvitelanding?user=88470b98-4248-e011-889f
-c965259bf186&signedParameters=Awyk_ENG2JjVVLMZL35lXZgVNfKVdBik-cdYXe_prRs.eyJlbW
FpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM
2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYt
Yzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc
3N1ZWRfYXQiOiIxMzcwMzkxODQ4IiwidmVyIjoiMSIsInAiOiIzIn0&cnId=88470b98-4248-e011-88
9f-c965259bf186&emailMessageId=fac8fa14-73cd-e211-be08-d229661189ee&emailType=Inv
iteFirstReminder&sdate=201306050024&utm_source=EmailInviteFirstReminder&utm_mediu
m=Email&sendercc=EG&emailLinkType=MainCallToAction>

reply+i-13574513-077aa914744a1748392fe83a8ef5edfd0acffc6a-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=8-l8UER91JieWys
jCkjh7JLNCkyt91oFJpJS7BmfBKI.eyJlbWFpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRh
MTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzcwMzkxODQ4IiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=fac8fa14-73cd-e
211-be08-d229661189ee&emailType=InviteFirstReminder&sdate=201306050024&utm_source
=EmailInviteFirstReminder&utm_medium=Email&sendercc=EG&emailLinkType=Unsubscribe>
.

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Comment by BeshoyAtef** [comment]
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ZftgOUwv_RJ
j7KJyGUQ1k7bcLv5cQCmO8rAt8t37NWw.eyJlbWFpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3
NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmc
m9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtYzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MG
Q1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc3N1ZWRfYXQiOiIxMzcwOTk2NzMzIiwidmVyIjoiMSI
sInAiOiIzIn0&emailMessageId=72662e48-f3d2-e211-884e-85d1397596ad&emailType=Invite
SecondReminder&sdate=201306120025&utm_source=EmailInviteSecondReminder&utm_medium
=Email&sendercc=EG&emailLinkType=Logo>

Hi,

Beshoy Atef is still waiting to connect with you.

<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ftux/emaili
nvitelanding?signedParameters=ZftgOUwv_RJj7KJyGUQ1k7bcLv5cQCmO8rAt8t37NWw.eyJlbWF
pbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2
YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtY
zk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc3
N1ZWRfYXQiOiIxMzcwOTk2NzMzIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=72662e48-f3d
2-e211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120025&utm_so
urce=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Profile
Thumb>

Beshoy Atef
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ftux/emaili
nvitelanding?signedParameters=ZftgOUwv_RJj7KJyGUQ1k7bcLv5cQCmO8rAt8t37NWw.eyJlbWF
pbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2
YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmcm9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtY
zk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MGQ1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc3
N1ZWRfYXQiOiIxMzcwOTk2NzMzIiwidmVyIjoiMSIsInAiOiIzIn0&emailMessageId=72662e48-f3d
2-e211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120025&utm_so
urce=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Profile
Name>

Invite sent: 29 May, 2013

Continue
<http://www.skillpages.com/i/ftux/emailinvitelanding?signedParameters=ZftgOUwv_RJ
j7KJyGUQ1k7bcLv5cQCmO8rAt8t37NWw.eyJlbWFpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3
NDRhMTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJmc
m9tIjoiODg0NzBiOTgtNDI0OC1lMDExLTg4OWYtYzk2NTI1OWJmMTg2IiwidG8iOiI0MTU3ZGEyNi05MG
Q1LTU4NTEtOTFlYi1jOWQxN2FiODQzMzMiLCJpc3N1ZWRfYXQiOiIxMzcwOTk2NzMzIiwidmVyIjoiMSI
sInAiOiIzIn0&emailMessageId=72662e48-f3d2-e211-884e-85d1397596ad&emailType=Invite
SecondReminder&sdate=201306120025&utm_source=EmailInviteSecondReminder&utm_medium
=Email&sendercc=EG&emailLinkType=MainCallToAction>

reply+i-13574513-077aa914744a1748392fe83a8ef5edfd0acffc6a-3515278@reply.github.co
m was invited to join SkillPages by Beshoy Atef. To stop receiving emails from
SkillPages click here
<http://www.skillpages.com/i/account/unsubscribe?signedParameters=3udLNdPjnnWvP6B
fsNQevRjKeg4xVtsiYGMu3brzZQ4.eyJlbWFpbCI6InJlcGx5K2ktMTM1NzQ1MTMtMDc3YWE5MTQ3NDRh
MTc0ODM5MmZlODNhOGVmNWVkZmQwYWNmZmM2YS0zNTE1Mjc4QHJlcGx5LmdpdGh1Yi5jb20iLCJpc3N1Z
WRfYXQiOiIxMzcwOTk2NzMzIiwidmVyIjoiMSIsInAiOiIwIn0&emailMessageId=72662e48-f3d2-e
211-884e-85d1397596ad&emailType=InviteSecondReminder&sdate=201306120025&utm_sourc
e=EmailInviteSecondReminder&utm_medium=Email&sendercc=EG&emailLinkType=Unsubscrib
e> .

© 2013 SkillPages, Blackrock Business Park, Dublin, Ireland and 228 Hamilton
Avenue, 3rd Floor, Palo Alto, CA 94301.

**Pull request merged**